modules/core/src/test/java/org/apache/ignite/platform/PlatformProcessUtils.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.platform;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.ignite.IgniteException;
+
+/**
+ * Platform process utils for tests.
+ */
+@SuppressWarnings("unused") // Called from Platform.
+public class PlatformProcessUtils {
+    /** */
+    private static volatile Process process;
+
+    /**
+     * Starts a new process.
+     *
+     * @param file Executable name.
+     * @param arg1 Argument.
+     * @param arg2 Argument.
+     * @param workDir Work directory.
+     * @param waitForOutput A string to look for in the output.
+     */
+    public static void startProcess(String file, String arg1, String arg2, String workDir, String waitForOutput)
+            throws Exception {
+        if (process != null)
+            throw new Exception("PlatformProcessUtils can't start more than one process at a time.");
```

Comment:
It would be great to return `PID` and then use it to kill the process. Unfortunately, `Process.pid()` was only added in Java 9, and we have to be able to run on Java 8.

So yes, this is a simplification - good enough for tests.